### PR TITLE
Add workaround to crash caused by image upload. (on live site)

### PR DESCRIPTION
**Pull request by neelbagora**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -123,6 +123,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("DATA");
```
Comment (avenkat-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:126]: nit: More useful print if you want to keep this?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -315,20 +316,26 @@ public String getUploadedFileUrl(HttpServletRequest request, String formInputEle
...
     // To support running in Google Cloud Shell with AppEngine's devserver, we must use the relative
     // path to the image, rather than the path returned by imagesService which contains a host.
     try {
+      System.out.println("Times");
       URL url = new URL(imagesService.getServingUrl(options));
       return url.getPath();
     } catch (MalformedURLException e) {
       return imagesService.getServingUrl(options);
-    }
+    } */
```
Comment (avenkat-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:338]: Do you need these comments?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (neelbagora) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:126]: Just a debugging print I forgot to remove, sorry about that.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (neelbagora) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:338]: Was going to uncomment this if the workaround did not work.

**Pull request merged**